servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java

```diff
@@ -148,9 +148,9 @@ public Long apply(Entry<Long, Double> input) {
       @Override
       public int compare(Entry<Long, Double> o1, Entry<Long, Double> o2) {
         double res = o1.getValue() - o2.getValue();
-        if (res < 0) {
+        if (Double.compare(res, 0) < 0) {
           return -1;
-        } else if (res > 0) {
+        } else if (Double.compare(res, 0) > 0) {
```

Comment:
I think you're right - `Comparator`s should only care about sign, not absolute value. I've changed it